src/Http/Client/FormDataPart.php

```diff
@@ -73,18 +73,27 @@ class FormDataPart
      */
     protected $_contentId;
 
+    /**
+     * The charset being used
+     *
+     * @var string|null
+     */
+    protected $_charset;
+
     /**
      * Constructor
      *
      * @param string $name The name of the data.
      * @param string $value The value of the data.
      * @param string $disposition The type of disposition to use, defaults to form-data.
+     * @param string|null $charset The charset of the data.
      */
-    public function __construct($name, $value, $disposition = 'form-data')
+    public function __construct($name, $value, $disposition = 'form-data', $charset = null)
```

Comment:
> The charset of the data

Could that possibly be a little misleading/confusing? One could think that this refers to the charset of the data part, ie the `charset` parameter for the `Content-Type` header, not to the possible global/default charset and that it affects the `Content-Disposition` header.

Reply:
`headerCharset` would match naming in `Email` and reduce misleading

Reply:
That's kinda what I had in mind, and the parameter should have a description that reflects that.